README.md

```
@@ -4,18 +4,115 @@
+# Run a specific test file
+cd client-web
+pnpm exec playwright test src/functional-e2e/path/to/file.spec.ts
+
+# Open Playwright UI mode (interactive test runner)
+cd client-web
+pnpm exec playwright test --ui
+```
+
+# Known issues
+
+### e2e tests
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)